net-analyzer/netdata/netdata-2.5.3.ebuild

```
@@ -39,47 +54,40 @@ RDEPEND="
 	net-misc/curl
 	net-misc/wget
 	sys-apps/util-linux
-	app-alternatives/awk
 	sys-libs/libcap
+	sys-libs/zlib:=
 	cups? ( net-print/cups )
-	app-arch/lz4:=
-	app-arch/zstd:=
-	app-arch/brotli:=
 	dbengine? (
 		dev-libs/judy
 		dev-libs/openssl:=
 	)
-	dev-libs/libpcre2:=
-	dev-libs/libuv:=
-	dev-libs/libyaml
-	dev-libs/protobuf:=
-	bpf? ( virtual/libelf:= )
-	sys-libs/zlib
 	ipmi? ( sys-libs/freeipmi )
-	dev-libs/json-c:=
+	jemalloc? ( dev-libs/jemalloc:= )
 	mongodb? ( dev-libs/mongo-c-driver )
```

```suggestion
	mongodb? ( dev-libs/mongo-c-driver:0 )
```

mongo-c-driver is (most likely) going to have a new slot, so this would be awesome to have to avoid future churn.

https://github.com/gentoo/gentoo/pull/43278